modules/rtp_rtcp/source/rtp_sender_egress.cc

```diff
@@ -144,6 +144,14 @@ void RtpSenderEgress::SendPacket(RtpPacketToSend* packet,
   // data after rtp header may be corrupted if these packets are protected by
   // the FEC.
   int64_t diff_ms = now_ms - packet->capture_time_ms();
+  int pktype = packet->packet_type()==RtpPacketMediaType::kAudio?0:(packet->packet_type()==RtpPacketMediaType::kVideo?1:-1);
```

Comment:
this line is too busy and not readable, how about split it to several lines to make it clear?